python/tvm/relay/op/strategy/x86.py

```diff
@@ -507,10 +507,26 @@ def matmul_strategy_cpu(attrs, inputs, out_type, target):
     return strategy
 
 
+def is_any(shape):
```

Comment:
`is_dynamic` might be a more appropriate name for this function.

Reply:
renamed to `is_dynamic_shape`